torch/csrc/autograd/python_variable.h

```diff
@@ -62,3 +66,7 @@ inline const at::Tensor& THPVariable_Unpack(PyObject* obj) {
 }
 
 TORCH_PYTHON_API c10::impl::PyInterpreter* getPyInterpreter();
+
+std::pair<py::object, py::dict> get_args_kwargs(const c10::OperatorHandle& op, const std::vector<c10::IValue>& arguments);
+
+void push_out_to_stack(const c10::OperatorHandle& op, torch::jit::Stack* stack, py::object out, const char* msg);
```

Comment:
The names here feel too generic; it's especially bad because this file is dumping the functions into top-level namespace (no sub-namespace!!) At the very least they should indicate something Python-related is going on. I suggest `parseIValuesToPyArgsKwargs` and `pushPyOutToStack`